love/src/Utils.test.js

```diff
@@ -20,6 +20,14 @@ describe('htmlToJiraMarkdown', () => {
       'This is a string with mixed content\r\nh1. This is a heading.\r\nThis is a [link|http://google.com/].\r\n';
     expect(htmlToJiraMarkdown(input)).toEqual(expectedOutput);
   });
+
+  it('should convert Jira tickets to links', () => {
+    const input =
+      '<p>This is a string with mixed content</p><h1>This is a heading.</h1><p>This is an already formatted link <a href="https://jira.lsstcorp.org/browse/DM-41184" rel="noopener noreferrer" target="_blank">DM-41184</a>.</p><p>This is a ticket name not yet formatted DM-41184.</p>';
+    const expectedOutput =
+      'This is a string with mixed content\r\nh1. This is a heading.\r\nThis is an already formatted link [DM-41184|https://jira.lsstcorp.org/browse/DM-41184].\r\nThis is a ticket name not yet formatted [DM-41184|https://jira.lsstcorp.org/browse/DM-41184].\r\n';
+    expect(htmlToJiraMarkdown(input)).toEqual(expectedOutput);
+  });
```

Comment:
Consider adding a third test case, where
```
[DM-41184|https://jira.lsstcorp.org/browse/DM-41184]
```
is already present to make sure that it remains formatted correctly.